support-data-rest/src/main/java/org/uniprot/api/subcell/service/SubcellularLocationSortClause.java

```diff
@@ -1,13 +1,15 @@
 package org.uniprot.api.subcell.service;
 
 import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
 import org.uniprot.api.rest.search.AbstractSolrSortClause;
 import org.uniprot.store.search.field.SubcellularLocationField;
 
 /**
  * @author lgonzales
  * @since 2019-07-19
  */
+@Service
```

Comment:
I think this is a component, right? Can you please check all the others Sort for this comment